duplicity-backup.sh

```diff
@@ -361,7 +361,7 @@ mailcmd_nail() {
   ${MAILCMD} -s "${EMAIL_SUBJECT}" -r "${EMAIL_FROM}" "${EMAIL_TO}" < "${LOGFILE}"
 }
 mailcmd_else() {
-  ${MAILCMD} "${EMAIL_SUBJECT}" "${EMAIL_FROM}" "${EMAIL_TO}" < "${LOGFILE}"
+  mailcmd_sendmail
```

Comment:
I'm not sure how you created your initial commit 0a4c428 but this one is wrong (the parent is my latest commit but it reverses my latest commit, probably a rebase and squash from your side where you discarded my change by mistake).

I'm gonna merge properly your commit 0501d233 instead of 0a4c428 in dev.